api/envoy/type/grpc/v2/grpc_status.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package envoy.type.grpc.v2;
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.type.grpc.v2";
+option java_outer_classname = "GrpcStatusProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.type.grpc.v3";
+
+// [#protodoc-title: GRPC status codes]
+
+// GRPC response codes supported.
+enum Status {
```

Comment:
No this doesn't exist. I don't think we should define this as enum but just as `int32` (that's what google.rpc.Status does) because in data plane there will be custom defined error codes as well.